objects/src/notes/assets.rs

```diff
@@ -35,13 +35,10 @@ impl NoteAssets {
     ///
     /// # Errors
     /// Returns an error if:
-    /// - The asset list is empty.
     /// - The list contains more than 256 assets.
     /// - There are duplicate assets in the list.
     pub fn new(assets: Vec<Asset>) -> Result<Self, NoteError> {
-        if assets.is_empty() {
-            return Err(NoteError::EmptyAssetList);
```

Comment:
Maybe this error should be deleted? Now that a note with no assets is allowed.

Reply:
good idea